node/cmd/guardiand/node.go

```diff
@@ -1353,6 +1352,9 @@ func runNode(cmd *cobra.Command, args []string) {
 		guardianOptions = append(guardianOptions, node.GuardianOptionBigTablePersistence(bigTableConnectionConfig))
 	}
 
+	// Finally, add the processor component
+	guardianOptions = append(guardianOptions, node.GuardianOptionProcessor())
```

Comment:
Help me understand why this is done here instead of just after line 1326.  I don't see any dependencies that would cause it.

Reply:
moved it to line 1327 like you suggested.